saleor/order/models.py

```diff
@@ -224,9 +225,11 @@ class Order(ModelWithMetadata):
     redirect_url = models.URLField(blank=True, null=True)
     objects = OrderQueryset.as_manager()
 
-    class Meta(ModelWithMetadata.Meta):
+    class Meta:
         ordering = ("-pk",)
         permissions = ((OrderPermissions.MANAGE_ORDERS.codename, "Manage orders."),)
+        indexes = [GinIndex(fields=["user_email"])]
+        indexes.extend(ModelWithMetadata.Meta.indexes)
```

Comment:
above I suggested approach without using extend, also apply it here if you think it's better.